API_usage.md

```diff
@@ -93,14 +93,21 @@ One you've logically grouped the datasets into collectionss, create dataset defi
   ],
   "discovery_items": [
     {
-      "discovery": "s3",
```

Comment:
this isn't needed anymore? I guess we haven't done CMR in a bit

Reply:
we've never done cmr :D